data/tabular/skin_reaction/meta.yaml

```diff
@@ -59,3 +61,86 @@ bibtex:
       title = {Predicting chemically-induced skin reactions. Part I: QSAR models of skin sensitization
       and their application to identify potentially hazardous compounds},
       journal = {Toxicology and Applied Pharmacology}
+templates:
+    - The {#molecule |!}{SMILES__description} {SMILES#} causes {skin_reaction#no &NULL}reaction with the skin.
+    - The molecule with the {SMILES__description} {#representation of |!}{SMILES#} is not {skin_reaction#no &NULL}{skin_reaction__names__adjective}.
```

Comment:
Needs fix, uses "not" and recoding: `not {skin_reaction#no &NULL}`

Reply:
fixed